### DISCUSSION TITLE
Automatically remove unnecessary `# noqa` comments

### DISCUSSION THREAD

**Discussion by its-monotype** [astral-sh/ruff#5572] [Q&A] [answered]
How to activate [yesqa](https://github.com/asottile/yesqa) to automatically remove unnecessary `# noqa` comments?

**Comment by charliermarsh** [comment, marked as answer]
You can use the `unused-noqa` rule (`RUF100`): https://beta.ruff.rs/docs/rules/unused-noqa/
